config/application.yml.default

```diff
@@ -192,6 +192,9 @@ invalid_gpo_confirmation_zipcode: '00001'
 # LexisNexis #####################################################
 # Instant Verify and Phone Finder Integrations
 lexisnexis_account_id: test_account
+###################################################################
+# LexisNexis DDP/ThreatMetrix #####################################
```

Comment:
I see why we moved since this file is alphabetized, but now the groupings are a little weird because e.g. `lexisnexis_instant_verify_timeout` is no longer under the "Instant Verify and Phone Finder Integrations" block.

I'd suggest either:

1. Get rid of the comment blocks
2. Rename the new configuration so that it's alphabetized with other ThreatMetrix keys, e.g. `lexisnexis_threatmetrix_authentication_policy`

Reply:
Now that we've renamed the configuration, can you move the comment block back to where it was?